### PR TITLE
symbol support

### PR DESCRIPTION
**Pull request by mensfeld**
This PR adds support for symbols similar to strings.

### CODE REVIEW

**Comment by mensfeld** [comment]
@dasch ping

File: lib/king_konf/decoder.rb [hunk 1]
```diff
@@ -17,6 +17,10 @@ def string(value, **)
       value
     end
 
+    def symbol(value, **)
+      value
```
Comment (dasch) [inline lib/king_konf/decoder.rb]: Doesn't this need to do `value.to_sym`?

Comment (mensfeld) [reply lib/king_konf/decoder.rb]: true. :man_facepalming: will adapt specs and fix this.

**Comment by dasch** [comment]
Sorry for missing this!

Can you add a spec for decoding a symbol?

**Comment by dasch** [comment]
Can you rebase on master? Sorry, I just realized that the list of valid types was copied in multiple places, so I put it into `lib/king_konf.rb`.

**Comment by mensfeld** [comment]
No worries. Refactor is always good.

**Comment by dasch** [comment]
❤️

**Pull request merged**